packages/ios_platform_images/README.md

```diff
@@ -0,0 +1,37 @@
+# IOS Platform Images
+
+A Flutter plugin to share images between Flutter and iOS.
```

Comment:
Talk about densities. Something like A Flutter plugin to load images with the right pixel density between Flutter and iOS, or some such

Reply:
done